src/components/core/pivotTable/tests/agGridUtils.spec.ts

```diff
@@ -75,22 +73,6 @@ describe("getGridIndex", () => {
     );
 });
 
-describe("cellRenderer", () => {
-    it("should escape value", () => {
```

Comment:
This test is now obsolete as the values coming to cellRenderer are sanitized already.